nvim/.config/nvim/lua/alpha/dap.lua

```diff
@@ -38,14 +38,21 @@ vim.fn.sign_define("DapLogPoint", { text = "💬", texthl = "", linehl = "", num
 
 -- vim.cmd("au FileType dap-repl lua require('dap.ext.autocompl').attach()")
 
--- TODO replace with mason for check if installed and get the path
 -- the binary does not work
-local php_debug_adapter = vim.fn.stdpath("data") .. "/mason/packages/php-debug-adapter/extension/out/phpDebug.js"
+local ok, php_debug_adapter = pcall(require("mason-registry").get_package, "php_debug_adapter")
```

Comment:
Thanks, I found that I was doing something wrong, the plugin explanation expects to run with command node and the second argument, the js, but mason package it as an executable, so I changed the command to `php-debug-adapter` and worked no problem. I don't need this, but help me to find the reason. What I would like is what you put on the comment to notify, and not configure if don't have it installed.